.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -21,7 +21,7 @@ oc apply -f ${script_dir}/smoke/${pod}.yaml || \
 # Check it eventually goes to 'running'
 #
 wait_time=600
-sleep_time=10
+sleep_time=5
```

Comment:
Optimizing the execution time of this script doesn't belong to this PR IMO.

Reply:
ack, it's a remnant from debugging. I'll put it into a different commit to have it revertable.